gsfpy/bindings.py

```diff
@@ -25,7 +25,21 @@
 
 _gsf_lib_rel_path = "libgsf/libgsf03-08.so"
 _gsf_lib_abs_path = path.join(path.abspath(path.dirname(__file__)), _gsf_lib_rel_path)
```

Comment:
These variable names should also be made consistent, `gsflib_...` or `libgsf_...`

Reply:
Done